pkg/v1/tkg/tkgpackageclient/package_install.go

```diff
@@ -32,6 +32,28 @@ const (
 
 // InstallPackage installs the PackageInstall and its associated resources in the cluster
 func (p *pkgClient) InstallPackage(o *tkgpackagedatamodel.PackageOptions, progress *tkgpackagedatamodel.PackageProgress, operationType tkgpackagedatamodel.OperationType) {
+	p.installPackage(o, progress, operationType)
+}
+
+// InstallPackageSync installs the PackageInstall and its associated resources in the cluster and returns an error if any
+func (p *pkgClient) InstallPackageSync(o *tkgpackagedatamodel.PackageOptions, operationType tkgpackagedatamodel.OperationType) error {
+	pp := getDefaultProcessConfig()
+
+	go p.installPackage(o, pp, operationType)
+
+	initialMsg := fmt.Sprintf("Installing package '%s'", o.PackageName)
+	if err := DisplayProgress(initialMsg, pp); err != nil {
+		if err.Error() == tkgpackagedatamodel.ErrPackageAlreadyExists {
+			log.Infof("Updated installed package '%s'", o.PkgInstallName)
+			return nil
+		}
+		return err
+	}
+
```

Comment:
You need another log message here for saying the package actually got installed:
```
log.Infof("\n %s", fmt.Sprintf("Added installed package '%s'",
		o.PkgInstallName))
```

Reply:
```suggestion
log.Infof("\n %s", fmt.Sprintf("Added installed package '%s'", o.PkgInstallName))
```